science-platform/values-idfint.yaml

```diff
@@ -0,0 +1,40 @@
+environment: idfint
+
+argo:
+  enabled: true
+cert_issuer:
+  enabled: true
+cert_manager:
+  enabled: true
+chronograf:
+  enabled: false
+exposurelog:
+  enabled: false
+gafaelfawr:
+  enabled: true
+influxdb:
+  enabled: false
+kapacitor:
+  enabled: false
+landing_page:
+  enabled: true
+logging:
+  enabled: false
+mobu:
+  enabled: true
+nginx_ingress:
+  enabled: true
+nublado:
+  enabled: true
+obstap:
+  enabled: true
+portal:
+  enabled: true
+postgres:
+  enabled: true
+squash_api:
+  enabled: false
+tap:
+  enabled: true
```

Comment:
Not sure about this either.  With no qserv this can't work, but maybe we just want the endpoint up so we can do a health check?  If there is actually a qserv and I'm just ill informed, ignore me.

Reply:
I mean, maybe?  I just copied dev.  I presume that there will eventually be qserv in each?